docs/lib/datastore/fragments/android/getting-started/30_platformIntegration.md

```diff
@@ -1,2 +1,34 @@
-<!-- TODO: add Android gradle tools instructions -->
-<!-- shorter version of the one in the tutorial -->
\ No newline at end of file
+The `amplify-tools-gradle-plugin` Gradle plugin provides an integration layer between the Amplify CLI headless mode (i.e. no AWS credentials needed) and Android Studio. It integrates into the build process of the IDE to provide a seamless experience.
+
+First, add the plugin to your project-level `build.gradle`.
+
+Add the plugin as a dependency under the `buildscript`'s `dependencies` block:
+
+```groovy
+buildscript {
+    repositories {
+        mavenLocal()
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:4.0.0'
+        // Add this next line:
+        classpath 'com.amplifyframework:amplify-tools-gradle-plugin:1.0.1'
```

Comment:
are you guys updating the version on every release? On iOS we decided to drop the version on documentation, it makes our lives way easier.

ps: no need to address it now, I'm just curious.

Reply:
Hm. Yea, we can use `:1.+` or something. It's not a best practice, but it would keep our documentation correct, more-of-the-time.